examples/resources/mondoo_integration_aws_serverless/resource.tf

```diff
@@ -65,12 +71,13 @@ resource "mondoo_integration_aws_serverless" "aws_serverless" {
 # for single account deploys
 resource "aws_cloudformation_stack" "mondoo_stack" {
   name         = "mondoo-stack"
-  template_url = "https://s3.amazonaws.com/mondoo.${data.aws_region.current.name}/mondoo-lambda-stackset-cf.json"
+  template_url = "https://s3.amazonaws.com/mondoo.${data.aws_region.current.name}/mondoo-root-cf.json"
```

Comment:
Can we make it so that we include the property in the source code and leave this as an optional attribute to overwrite the internal one. Then we would probably introduce a aws region property but we can handle the rest then. Allows us to easily fix and update urls going forward without user requirement to use update the value.